### PR TITLE
Guzzle 7 support

### PR DESCRIPTION
**Pull request by Trixpua**

### CODE REVIEW

**Comment by mtownsend5512** [comment]
+1 - would like to see this added to the package, please

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         "php": ">=5.5.0",
         "meng-tian/php-async-soap": "~1.0",
         "meng-tian/soap-http-binding": "~0.2.0",
-        "guzzlehttp/guzzle": "^6.1"
+        "guzzlehttp/guzzle": "^6.1|^7.0.1"
```
Comment (realmtian) [inline composer.json:16]: Shouldn't it be `^6.1 || ^7.0.1`?

**Review by realmtian — COMMENTED** [review]
Thanks for the contribution, but I am not sure if `|` is the correct syntax to use.

**Comment by realmtian** [comment]
I've added support for guzzle 7 in the latest [release](https://github.com/meng-tian/async-soap-guzzle/tree/v0.4.0).

**Pull request closed without merging**